doc/changes/changes_0.1.0.md

```diff
@@ -24,6 +24,7 @@ Code name:
 * #98: Added more robust connection protocol
 * #99: Added multi/node udf discovery
 * #100: Added combined global and local UDF discovery
+* #177: Integrated improvements from temporary branches
```

Comment:
We should call this differently

Reply:
I propose: 
Added proxies for UDFs and Connections.

Reply:
yep sounds good